src/client/pages/OfferNew/Introduction/HeroOfferDetails.tsx

```diff
@@ -1,75 +1,84 @@
 import { colorsV3 } from '@hedviginsurance/brand'
-import { Address } from 'data/graphql'
 import { OfferData } from 'pages/OfferNew/types'
 import { useTextKeys } from 'utils/textKeys'
-import { LARGE_SCREEN_MEDIA_QUERY } from 'utils/mediaQueries'
-import { getHouseholdSize } from '../utils'
+import {
+  LARGE_SCREEN_MEDIA_QUERY,
+  MEDIUM_SCREEN_MEDIA_QUERY,
+} from 'utils/mediaQueries'
+import { getHouseholdSize, quoteDetailsHasAddress } from '../utils'
+import { getAddress } from '../Checkout/InsuranceSummaryDetails'
 
 type Props = {
   offerData: OfferData
 }
 
-type QuoteWithStreet = {
-  quoteDetails: { street: Address['street'] }
-}
-
-const Container = styled.div`
-  padding: 2rem 0 1rem;
+const Wrapper = styled.div`
+  padding: 2.5rem 0 1rem;
   color: ${colorsV3.white};
 
   ${LARGE_SCREEN_MEDIA_QUERY} {
     padding-top: 0;
   }
 `
-
-const Heading = styled.h1`
+const Headline = styled.h1`
   margin: 0;
   text-transform: uppercase;
-  font-size: 0.75rem;
+  font-size: 1rem;
+`
+const OfferInfoWrapper = styled.div`
+  padding: 1rem 0;
 
   ${LARGE_SCREEN_MEDIA_QUERY} {
-    font-size: 1rem;
+    padding-top: 0;
   }
 `
-
-const OfferInfo = styled.div`
+const NameAndCoInsured = styled.div`
   font-size: 2rem;
-  word-break: break-all;
 
-  ${LARGE_SCREEN_MEDIA_QUERY} {
+  ${MEDIUM_SCREEN_MEDIA_QUERY} {
     font-size: 3rem;
   }
 `
+const Address = styled.div`
+  font-size: 1.375rem;
+
+  ${MEDIUM_SCREEN_MEDIA_QUERY} {
+    font-size: 2.5rem;
+  }
+`
 
 export const HeroOfferDetails: React.FC<Props> = ({ offerData }) => {
   const [numberCoInsured, setNumberCoInsured] = useState<number | null>(null)
-  const [street, setStreet] = useState<Address['street'] | null>(null)
+  const [address, setAddress] = useState<string | null>(null)
 
   const { person, quotes } = offerData
 
   useEffect(() => {
     const householdSize = getHouseholdSize(quotes[0].quoteDetails)
     setNumberCoInsured(householdSize - 1)
 
-    const quoteWithStreet = quotes.find((quote) => {
-      return 'street' in quote.quoteDetails
+    const quoteWithAddress = quotes.find((quote) => {
+      return quoteDetailsHasAddress(quote.quoteDetails)
     })
-    if (quoteWithStreet) {
-      setStreet((quoteWithStreet as QuoteWithStreet).quoteDetails.street)
+    if (quoteWithAddress) {
+      const address = getAddress(quoteWithAddress.quoteDetails)
+      setAddress(address)
     }
   }, [quotes])
 
   const textKeys = useTextKeys()
   return (
-    <Container>
-      <Heading>{textKeys.HERO_OFFER_DETAILS_HEADER()}</Heading>
-      <OfferInfo>
-        {person.firstName}
-        {!!numberCoInsured && ` +${numberCoInsured}`}
-      </OfferInfo>
-      {street && <OfferInfo>{street}</OfferInfo>}
-    </Container>
+    <Wrapper>
+      <Headline>{textKeys.HERO_OFFER_DETAILS_HEADER()}</Headline>
+      <OfferInfoWrapper>
+        <NameAndCoInsured>
+          {person.firstName}
+          {Boolean(numberCoInsured) && ` +${numberCoInsured}`}
```

Comment:
With the suggestion above we can simplify this:
```suggestion
          {numberCoInsured > 0 && ` +${numberCoInsured}`}
```

Reply:
Yep that's more reasonable 👍

Reply:
Oh, now I realised why I did it the way I did it - `numberCoInsured` can also be `null`

Reply:
Aha! I'm good with using the double `!!` then. Or do we feel that it's too unclear to use that?

Reply:
Right, but only if we are setting it up with some hooks state, no?

If we derive state directly from the offer data, it will always be defined I think

Reply:
Yep, this will soon be gone.

But when comparing `!!` and explicit type coercion with `Boolean()` I think the latter is more clear.